app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  def self.last_posts(user)
+    return unless user
+
+    Post.where(author_id: user[:id]).order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] the user has many relationship with post, like and comment should be set in this class